strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/StorageApiTestConfig.java

```diff
@@ -18,7 +18,8 @@
           EventsConfig.class,
           StorageCoreConfig.class,
           StorageApiConfig.class,
-          NullLayoutConfiguration.class
+          NullLayoutConfiguration.class,
+          StorageMultipartConfig.class
```

Comment:
@JanBucko you could add this only to `StorageApiConfig` instead of doing it in all config classes.

Reply:
Done :) this also fixed the issue with bean creation inside unit test that i mentioned, thank you :) this solution came up to me but I was afraid it would create circular dependency and didnt even test it, now I realise the dependent Config file is created before, silly me.

Reply:
Also I shall be able to write some tests for it soon :)